src/components/Calculator.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable react/button-has-type */
+import React, { Component } from 'react';
+import './Calculate.css';
+
+const createDigit = () => {
+  const digits = [];
+  for (let i = 9; i >= 1; i -= 1) {
+    digits.push(<button key={i}>{i}</button>);
+  }
+  return digits;
+};
+// eslint-disable-next-line react/prefer-stateless-function
```

Comment:
- [ ] Kindly avoid disabling linters manually, please remove this line as there should be no custom change to the linters 👍🏿